code/modules/admin/IsBanned.dm

```diff
@@ -58,7 +58,7 @@
 		// If there is no row, they didnt accept
 		if(!check_query.NextRow())
 			qdel(check_query)
-			return list("reason"="tos", "desc"="\nReason: You are trying to connect without accepting server ToS. If this error persists, please contact the server host.")
+			return list("reason"="tos", "desc"="\nReason: You are trying to connect without accepting server ToS. If you did not get a ToS popup, please go to paradise13.org/fixtos")
```

Comment:
This URL here is a shortened link to a page on the forums, which automatically takes the client IP and tells the router to chuck them at the ToS server.

I didnt make it a config option because this is never going to change. The page itself also includes support info.
![image](https://user-images.githubusercontent.com/25063394/154683193-0c5b4ac7-c0ca-4c65-b4c2-47257898d8d2.png)